README.md

```diff
@@ -4,19 +4,19 @@
 
 I2Cwrapper is a **generic modular framework for Arduino I2C target devices**. It supports AVR-based Arduinos (Uno, Nano etc.), ESP8266, and ESP32 platforms. 
 
-I2Cwrapper consists of an easily extensible firmware framework and a controller library. Together, they **take care of the overhead** necessary for implementing an I2C target device and let you concentrate on implementing the actual target functionality. Ready to use modules exist for **stepper motor** control, **servo motor** control, **digital/analog pin** control, and **ESP32 touch button** control (see below).
+I2Cwrapper consists of an easily extensible firmware framework and a controller library. Together, they **take care of the overhead** necessary for implementing an I2C target device and let you concentrate on implementing the actual target functionality. <replace ith Available Modules section?> Ready to use modules exist for **[stepper motor](#accelstepperi2c)** control via `AccelStepper`, **[servo motor](#servoi2c)** control via [`Servo` library](https://www.arduino.cc/reference/en/libraries/servo), **[digital/analog pin](#pini2c)** control, and **[ESP32 touch button](#esp32sensorsi2c)** control (see below).
 
 [Download I2Cwrapper on github.](https://github.com/ftjuh/I2Cwrapper)
 
 [I2Cwrapper library documentation](https://ftjuh.github.io/I2Cwrapper/)
 
 Note that I2Cwrapper uses the [current I2C terminology](https://www.nxp.com/docs/en/user-guide/UM10204.pdf) which replaced *master* with *controller*, and *slave* with *target*.
 
-## Basic components
+## Basic Components
 
 The I2Cwrapper framework consists of **four basic components.** The first two drive the **I2C target device**:
 
-1. A **firmware framework** for the target device (`firmware.ino`). It provides the basic I2C target functionality:
+1. A **firmware framework** for the target device [(`firmware.ino`)](firmware/firmware.ino). It provides the basic I2C target functionality:
```

Comment:
I'll test these links in the doxygen .html's once I know how to build.

The advantage of "relative" links is that they work on your development instance. The disadvantage is if the README(.html) gets detached.

Reply:
My priority would be to have the links working at the github main page, that's also the behavior I'd expect from others' repositories.

(Unfortunately, it seems that besides the relative/absolute issue, github treats internal links with anchors different from my local editor (Typora), but I see you already fixed those.)

Reply:
> I simply ran `doxygen` from the project's root. I got a bunch of documentation, but it did not include "classes" in the table-of-contents. Any idea what to do?

Another issue is your docs do not interpret the html `<del>` tag I used to mark (yet document) completed todos (compare https://ftjuh.github.io/I2Cwrapper/todo.html and https://awgrover.github.io/I2Cwrapper/todo.html). 
I just created the docs from scratch from command line and see no problems. Maybe a version thing, I see you're using doxygen 1.8.13, my version is 1.8.17? Have you looked at the doxygen messages if they threw any warnings or errors? 

> * do not handle relative links to files, like `src/ESP32sensorsI2C.h`
> Shall I use absolute links to files (should work in github & doxygen)?

In my version, they link to the doxygen documentation of that file that is linked in the "Files/File List" submenu, which I consider the desired behavior. So no absolute links.

> Shall I abandon links to markdown headers (since it doesn't work in doxygen)?

It seems doxygen [is interpreting the anchor links as @ref links](https://www.doxygen.nl/manual/markdown.html#md_inlinelinks), which feels wrong, but alas. I'll have to see if there is a better way, but for the moment I can live with them working on the github front page and not working inside the doxygen documentation.

However, I realize that for them  to work on the  github front page, the whitespaces have to be replaced by hyphens like this:

`[adjustable minimum delay](#Adjusting-the-I2C-delay)`

Reply:
checked `[#` links and fixed

Reply:
Built doxygen files with doxygen 1.8.18. The warnings about \<del> went away. The table-of-contents-side-panel looks much better. I've committed the new docs/ directory with this pull-request.

Current warnings:

    warning: Tag 'TCL_SUBST' at line 270 of file 'Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
    warning: Tag 'CLANG_ASSISTED_PARSING' at line 1131 of file 'Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
    warning: Tag 'CLANG_OPTIONS' at line 1139 of file 'Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
    warning: Tag 'CLANG_DATABASE_PATH' at line 1149 of file 'Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.